metacat-common/src/main/java/com/netflix/metacat/common/dto/TableDto.java

```diff
@@ -48,7 +48,7 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class TableDto extends BaseDto implements HasDataMetadata, HasDefinitionMetadata {
+public class TableDto extends BaseDto implements HasDataMetadata, HasDefinitionMetadata, HasBusinessMetadata {
```

Comment:
This is so weird cause it' implements all these interfaces by default. Again I understand this is the pattern but we should break it.